src/main/java/Duke.java

```diff
@@ -1,10 +1,474 @@
+            System.out.println(numOfInfo + ". " + storedTask.toString());
+            numOfInfo++;
+        }
+    }
+
+    public static void displayCurrentTask(ArrayList<Task> storedTasks,
+                                          Integer taskIndex, boolean isAdd) {
+
+        if(isAdd) {
+            System.out.println(HORIZONTAL_LINE + ADD_MESSAGE_RECEIVED_MESSAGE);
+        } else {
+            System.out.println(HORIZONTAL_LINE + DELETE_MESSAGE_RECEIVED_MESSAGE);
+        }
+
+        try {
+            System.out.println("\t" + storedTasks.get(taskIndex).toString());
+            int taskNumber = (isAdd) ? storedTasks.size() : storedTasks.size() - 1;
+            System.out.println("Now you have " + taskNumber + " task"
+                    + ((taskNumber > 1) ? "s" : ""));
+        } catch (IndexOutOfBoundsException e) {
+            printNoTaskWarning();
+        }
+        System.out.print(HORIZONTAL_LINE);
+
+        appendFile(storedTasks);
+    }
+
+    public static void appendFile(ArrayList<Task> storedTasks) {
+
+        try {
+            appendToFile(String.format("%s\n", storedTasks.get(storedTasks.size() - 1).toString()));
+        } catch (IOException e) {
+            printIOWarning(e);
+        }
+    }
+
+    public static String[] extractWords(String response)
+            throws NoTaskTimeException, NoTaskNameException {
+
+        int endIndexOfTask = (response.contains("/")) ?
+                response.indexOf("/") : response.length();
+        int beginIndexOfTask = response.indexOf(" ");
+
+        String taskName = response.substring(beginIndexOfTask, endIndexOfTask).trim();
+
+        String taskTime = response.substring(endIndexOfTask).split(" ", 2)[1];
+
+        if (taskTime.isEmpty()) {
+            throw new NoTaskTimeException();
+        } else if (taskName.isEmpty()) {
+            throw new NoTaskNameException();
+        }
+
+        return new String[]{taskName, taskTime};
+    }
+
+    public static void addDeadline(ArrayList<Task> storedTasks,
+                                   String[] arrOfTaskAndTime, boolean isFromFile) {
+
+        Deadline t = new Deadline(arrOfTaskAndTime[0], arrOfTaskAndTime[1]);
+        storedTasks.add(t);
+        if (!isFromFile) {
+            displayCurrentTask(storedTasks,storedTasks.size() - 1, true);
+        }
+    }
+
+    public static void addEvent(ArrayList<Task> storedTasks,
+                                String[] arrOfTaskAndTime, boolean isFromFile) {
+
+        Event t = new Event(arrOfTaskAndTime[0], arrOfTaskAndTime[1]);
+        storedTasks.add(t);
+        if (!isFromFile) {
+            displayCurrentTask(storedTasks, storedTasks.size() - 1, true);
+        }
+    }
+
+    private static void appendToFile(String textToAppend) throws IOException {
+
+        FileWriter fw = new FileWriter(Duke.FILE_PATHWAY, true);
+        fw.write(textToAppend);
+        fw.close();
+    }
+
+    public static void printIllegalCommandWarning() {
```

Comment:
Most of these functions are simply adding `HORIZONTAL_LINE` before and after string constants that are initialized at the very beginning. Perhaps we could simply add them to those string constants (since they are not used elsewhere) and simply print that (modified) string constant instead of calling a function.
Another way, perhaps is to write a function that takes in a string and simply prints it out. This way, you can recycle that function multiple times instead of having to write these functions that repeats itself in terms of functionality.